### DISCUSSION TITLE
TLS handshake timeout When docker-compose

### DISCUSSION THREAD

**Discussion by amons000** [Budibase/budibase#2928] [Help]
<img width="885" alt="image" src="https://user-images.githubusercontent.com/35733429/136532473-30db2cfe-d3e9-4396-9156-008272b72f06.png">

I have configured docker's proxy and curl's proxy. But when I get ` https://registry-1.docker.io/v2/`, it still show some error, seems like I still have some proxy setting undone.

**Comment by shogunpurple** [comment]
Hey @Edwardlin-zlt - seems like there might be a problem with `scarf`, our analytics provider for docker. Can you update your images in `docker-compose.yaml` to be:

`budibase/apps`
and
`budibase/worker` 

instead of the scarf URLs?

Thanks!

**Reply by amons000** [reply]
<img width="882" alt="image" src="https://user-images.githubusercontent.com/35733429/136538537-fdf0222a-ccf1-4ab9-87ba-da621686464d.png">
Hah... Maybe it's my network issue. Some of them successed, some not.
